src/inspector/models/general/appearance/appearancesettingsmodel.cpp

```diff
@@ -96,6 +99,8 @@ void AppearanceSettingsModel::requestElements()
         }
     }
     m_elementsForOffsetProperty = elementsForOffsetProperty.values();
+
+    m_elementsForColorProperty = QList<EngravingItem*>(selection()->elements().begin(), selection()->elements().end());
```

Comment:
I'm afraid it's not safe to just take all the selected elements. This may contain things like InstrumentNames and Brackets, and those are not really safe to work with in the Inspector. See also `ElementRepositoryService::exposeRawElements`, where such elements are filtered.

A better solution, albeit slightly more difficult perhaps, would be to revise the fact that `ElementRepositoryService::exposeRawElements` (and thus `ElementRepositoryService::takeAllElements()`) includes all Chords when a Beam is selected. I don't see why that's desirable, and I do think it's causing trouble here.

So, what I would propose is:
1. Check where `takeAllElements` is used, and verify whether it is not harmful to remove those extra `Chords` from it
2. Remove those lines from `exposeRawElements` that add those extra chords
3. Check how this affects other methods inside `ElementsRepositoryService` that use `m_exposedElementList`. In cases where this changes the behaviour of those methods, consider whether adjustments need to be made to counter that change, or that the new behaviour is an improvement. 

And ideally, report your findings of each step here, because that makes it much easier to provide proper testing and feedback. Thanks!